### PR TITLE
added model files to db/models these have not yet been tested

### LINKED ISSUE #26: implement model files
I will be adding test files in the next stage as I build out the helper methods.

### PR DESCRIPTION
**Pull request by theenderweggin**
* finished added model files to the model folder
* next step will be adding helper functions 
* followed by writing tests to test these functions

closed #26

### CODE REVIEW

File: db/models/auths.js [hunk 1]
```diff
@@ -4,8 +4,8 @@ const bcrypt = Promise.promisifyAll(require('bcrypt-nodejs'));
 
 const Auth = db.Model.extend({
   tableName: 'auths',
-  profile: function() {
```
Comment (iMobs) [inline db/models/auths.js]: Did you change the Profile model to User too?

Comment (theenderweggin) [reply db/models/auths.js]: sorry your correct thats a mistake from when I changed everything to users. I revered the changes in most places. it looks like i missed that one. I'll correct changes and make a new pull request

Comment (iMobs) [reply db/models/auths.js]: Don't need a new PR, just push to the same branch.

Comment (theenderweggin) [reply db/models/auths.js]: I have correct the typo

**Pull request merged**